_dashboards/index.md

```diff
@@ -11,15 +11,56 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# Getting started with OpenSearch Dashboards
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is the preferred visualization platform for analyzing data in OpenSearch. It serves as the user interface for many openSearch features, including security, alerting, index state management, SQL, and more.
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and OpenSearch provides a full suite of plugins that you can use to monitor events, identify trends, and automate recurring activities.
```

Comment:
Do we need to differentiate that OpenSearch Dashboards has its own plugins? We should probably link to the topic in the related links.

Reply:
Added links under related topics

Reply:
I think it depends on the audience for this doc and which usage of "OpenSearch" we're thinking of here. The OpenSearch project/organization does provide a full suite of plugins; some of these are OpenSearch Dashboards plugins (front-end only), some are OpenSearch plugins (backend only), and some that require both types of plugins installed together to work correctly (e.g. Security). That feels like a whole can of technical implementation worms to open here if the intended audience won't care.